examples/cogs.py

```diff
@@ -0,0 +1,18 @@
+import voltage  # Import voltage.
+from voltage.ext import commands  # Importing the cog framework so we can use the cogs.
```

Comment:
Cog framework? :trollface:

Reply:
`from voltage.ext.commands import Cog` :bowtie:

Reply:
# ***C O G  F R A M E W O R K***

I suppose.

Reply:
> # _**C O G F R A M E W O R K**_
> I suppose.

stop saying `I suppose` please.

Reply:
Brun ok. ILL CHANGE IT TO COMMAND FRAMEWORK FFS u guys

Reply:
> > # _**C O G F R A M E W O R K**_
> > I suppose.
> 
> stop saying `I suppose` please.

No I suppose :trollface: